test/test_utils/eks.py

```diff
@@ -215,10 +228,29 @@ def delete_cfn_stack_and_wait(stack_name):
         LOGGER.error(f"Error: Cannot delete stack: {stack_name}. Full Exception:\n{e}")
         describe_cfn_stack_events(stack_name)
 
+def delete_oidc_provider(eks_cluster_name):
+    """Function to delete the oidc provider created by kubeflow
+    Args:
+        eks_cluster_name: str
+    """
+    eks_client = init_eks_client()
+    iam_client = init_iam_client()
+    account_id = os.getenv("ACCOUNT_ID")    
+
+    try:
+        response = eks_client.describe_cluster(name=eks_cluster_name)
```

Comment:
Wouldn't you have already deleted the cluster at this point in the `delete_eks_cluster` function?

Reply:
True, I was waiting for CDK changes to catch this. Made the changes